src/main/java/com/diamondfire/helpbot/bot/command/impl/other/tag/Tag.java

```diff
@@ -0,0 +1,98 @@
+package com.diamondfire.helpbot.bot.command.impl.other.tag;
+
+import com.google.gson.JsonObject;
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.entities.*;
+
+public class Tag implements Cloneable {
+    
+    private String activator;
+    private String title;
+    private String response;
+    private long authorId;
+    private String image;
+    
+    public Tag(String activator, String title, String response, long authorId, String image) {
+        this.activator = activator;
+        this.title = title;
+        this.response = response;
+        this.authorId = authorId;
+        this.image = image;
+    }
+    
+    public String getActivator() {
+        return activator;
+    }
+    
+    public void setActivator(String activator) {
+        this.activator = activator.replaceAll(" ", "");
+    }
+    
+    public String getTitle() {
+        return this.title;
+    }
+    
+    public void setTitle(String title) {
+        this.title = title;
+    }
+    
+    public String getResponse() {
+        return this.response;
+    }
+    
+    public void setResponse(String response) {
+        this.response = response;
+    }
+    
+    public long getAuthorId() {
+        return this.authorId;
+    }
+    
+    public void setAuthorId(long authorId) {
+        this.authorId = authorId;
+    }
+    
+    public String getImage() {
+        return this.image;
+    }
+    
+    public void setImage(String image) {
+        this.image = image;
+    }
+    
+    public void set(TagProperty property, String newValue) {
+        if (property == TagProperty.ACTIVATOR) setActivator(newValue);
+        else if (property == TagProperty.TITLE) setTitle(newValue);
+        else if (property == TagProperty.RESPONSE) setResponse(newValue);
+        else if (property == TagProperty.IMAGE) setImage(newValue);
```

Comment:
Use an abstract method in the enum. Where it's like TagProperty.IMAGE.edit(newValue);